src/app/page.tsx

```diff
@@ -15,6 +16,8 @@ export default async function Home() {
         <h1>WörterWald</h1>
         <Hamburger />
       </header>
+
+      <LetterGrid />
```

Comment:
Why not already use letters from the game? game is fetched on top and already contains letters and mainLetter